hathor/verification/transaction_verifier.py

```diff
@@ -47,10 +49,84 @@ def verify(self, tx: Transaction, *, reject_locked_reward: bool = True) -> None:
         if tx.is_genesis:
             # TODO do genesis validation
             return
+        self.verify_without_storage(tx)
+        self.verify_sigops_input(tx)
+        self.verify_inputs(tx)  # need to run verify_inputs first to check if all inputs exist
+        self.verify_parents(tx)
+        self.verify_sum(tx)
+        if reject_locked_reward:
+            self.verify_reward_locked(tx)
+
+    def verify_unsigned_skip_pow(self, tx: Transaction) -> None:
+        """ Same as .verify but skipping pow and signature verification."""
+        tx.verify_unsigned_skip_pow()
+
+    @staticmethod
+    def verify_parents_basic(tx: Transaction) -> None:
```

Comment:
Why is it a static method?

Reply:
Changed in https://github.com/HathorNetwork/hathor-core/pull/797/commits/105e0598363e0125e9f6112ec72b213889790b61

Reply:
Thanks!